### PR TITLE
Feature/add sorting and filtering for users

### PR DESCRIPTION
**Pull request by viktoriia-horobets**
- [x]  Add sorting for each field in the table
- [x]  Add filtering for each field in the filter
- [x]  Fix styles for the filter
- [x] Add filter for articles

Use gem ransack for the task
https://trello.com/c/9Y8Yvmr4/41-add-sorting-to-each-field-in-model-for-admin-user
https://trello.com/c/OtXmp7zH/43-add-filtering-to-each-field-in-model-for-admin-user

https://user-images.githubusercontent.com/64083469/188325859-a6c8c030-2bdd-4d2c-9f70-2d4c16940469.mp4

https://user-images.githubusercontent.com/64083469/188328308-1c5a3cc7-4a95-48fe-b16d-b34a641962e0.mp4

### CODE REVIEW

File: app/javascript/packs/scripts/filter.js [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+  init()
+});
+
+function init() {
+  let elems = document.querySelectorAll('.dropdown-trigger-1');
   options = {
     closeOnClick: false,
     constrainWidth: false,
   };
+  console.log(elems)
```
Comment (mmrshk) [inline app/javascript/packs/scripts/filter.js:15]: ? :)

File: app/views/admins/_filter.html.erb [hunk 2]
```diff
@@ -1,79 +1,79 @@
...
+
+        <% end %>
        </form> 
      </div>  
    </div>  
-</div>
\ No newline at end of file
+  
+</div>
+
```
Comment (mmrshk) [inline app/views/admins/_filter.html.erb:79]: Remove extra line

File: config/2_5447532790767687998.enc [hunk 3]
```diff
@@ -0,0 +1 @@
+RXMOfyNTvr0lXrmWALIKQmotbZABNXk/roprKBd2lOWlVdfAT+/Bh95mtRltz5I3TOp2KTDa/zUePXkgrWlLd/neMGnAVsjIN5C9lll5t9/78zX28P5IwfYXErJmxm2wQB6PqtpQPQi3s0BH9xZIRXc/IoThG95o0yf7HdUpD/w4SL3B/z7JtKr4mAUrQ9QAZBUWyP9zluWbBht0EHOTfJ2A/Kax2/CTNAXn/cA9+v5Af0yrqi8FAtMmz/F/MaoVsc4eFx8ALtQCXeOEcTV6OkZ5V0mJSzS8HxY4q/gB90ViJT1Csup4RIFSeIVI84cw70eLaedtKOqTWBmXfTNcPwQXVgCbJFngku1f6RRL1ILWo5Lk0cypAeEULvJAkLwwpp3gxkb1nJIrXm662ST6crsovBaxbepK17J3--VKHUd1Whi9s2IaFM--DksbqcOntSS6y9MqAU2BZg==
```
Comment (mmrshk) [inline config/2_5447532790767687998.enc:1]: What is this?

**Review by mmrshk — CHANGES_REQUESTED** [review]
Fix comments and remove coverage files.

File: app/javascript/packs/scripts/filter.js (hunk 1 above)
Comment (viktoriia-horobets) [reply app/javascript/packs/scripts/filter.js]: fixed

File: app/views/admins/_filter.html.erb (hunk 2 above)
Comment (viktoriia-horobets) [reply app/views/admins/_filter.html.erb]: done

**Pull request merged**